share/revng/test/tests/analysis/Decompilation/clang/linked-lists.c.filecheck

```diff
@@ -4,8 +4,8 @@ This file is distributed under the MIT License. See LICENSE.md for details.
 The init_list function should have the proper detected return type, a pointer to
 struct representing the node of the linked list.
 
-CHECK: [[NODE_STRUCT:struct_[0-9]+]] *init_list(void)
-
+CHECK: [[NODE_STRUCT:struct_[0-9]+]] *function_0x{{[0-9a-f]+}}_Code_x86_64(void)
+0x400840_Code_x86_64
```

Comment:
Nonono, we do not harcode addresses in test ground truth

Reply:
The hard coded thing (as well as the commit message) is on purpose, to ensure it catches your attention and we don't lose this.

The point I wanted to bring up here is that we have to ban _all_ the `init_` names because as far is I understand there're no constraints on the `CSV` names - hence anything can come after it.

Should we come up with a better prefix for `init_rax` and friends? Hopefully one that no user would want to use. Or just change this test and let it be?

Reply:
> as far is I understand there're no constraints on the CSV names - hence anything can come after it

Mh, right. `undefined_${CSV}`? `revng_undefined_${CSV}`?

Reply:
> I prefer `revng_`, but either one works.

OKGO

Reply:
Done.